assignments/this.js

```diff
@@ -1,26 +1,69 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global - this will call the window of the browser because the binding of this is Global
```

Comment:
Generally, it is the "global object". Whatever that is depends on the environment. On browsers, it indeed is `window`, but in Node.js, a runtime environment used to run JS outside the browser, the global object is `console`.